<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>341</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>702</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>.z80 Extractor</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_pic">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>271</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>411</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Izlistavanje odabranog fajla:</string>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Putanja do izabranog fajla.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Ucitajte fajl</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Pokreni izlistavanje</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>411</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Izlistavanje svih fajlova u trenutnom direktorijumu:</string>
    </property>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Izlistaj sve</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Izlistavanje
programa</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Izlistavanje
promenljivih</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>702</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>...</string>
    </property>
    <addaction name="actionInfo"/>
    <addaction name="separator"/>
    <addaction name="actionIzlaz"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionInfo">
   <property name="icon">
    <iconset>
     <normalon>../../Downloads/ZX_Spect-whoami_d-1540/icon_info.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>O programu</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionIzlaz">
   <property name="icon">
    <iconset>
     <normalon>../../Downloads/ZX_Spect-whoami_d-1540/icon_exit.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Izlaz</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>checkBox</tabstop>
  <tabstop>checkBox_2</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton_4</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
